src/resources/OrganizationAccess/tests/OrganizationAccess.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import API from '../../../APICore';
+import Access from '../Access';
+
+jest.mock('../../../APICore');
+
+const APIMock: jest.Mock<API> = API as any;
+
+
+describe('OrganizationAccess', () => {
+    let organizationAccess: Access
+    const api = new APIMock() as jest.Mocked<API>;
+    const serverlessApi = new APIMock() as jest.Mocked<API>;
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+        organizationAccess = new Access(api, serverlessApi);
+
+    });
+
+    describe('getApiKeys', () => {
+        it('should make a GET call to the specific organization access url', () => {
+            const params = {
+                accessLevel: ['CUSTOM'],
+                privilegeOwner: 'PLATFORM',
+                privilegeTargetDomain: 'SOURCE',
+                organizationId: 'testorg',
+            }
+            organizationAccess.getApiKeys(params);
+            expect(api.get).toHaveBeenCalledTimes(1);
+            expect(api.get).toHaveBeenCalledWith(`${Access.baseUrl}/apikeys/accessLevel=CUSTOM&privilegeOwner=PLATFORM&privilegeTargetDomain=SOURCE'`);
+        })
+    })
+
+})
```

Comment:
idk know why but your IDE is not applying prettier on save/commit? 🤔 

I made one change in the file and a bunch of ; appeared , you might want to look into that 👀

Reply:
yup! I had to restart my IDE to trigger my prettier, it was not working as expected but should be good now 👍

Reply:
<img src="https://media4.giphy.com/media/WQr2txk5iEYUS6Kv3d/giphy.gif"/>